tests/src/dir.common.props

```diff
@@ -1,17 +1,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
 
- <!-- Set default Configuration and Platform -->
-  <PropertyGroup>
-    <__BuildArch Condition="'$(__BuildArch)' == ''">x64</__BuildArch>
-    <__BuildType Condition="'$(__BuildType)' == ''">Debug</__BuildType>
-    <__BuildOS Condition="'$(__BuildOS)' == ''">Windows_NT</__BuildOS>
-    <BuildArch>$(__BuildArch)</BuildArch>
-    <BuildType>$(__BuildType)</BuildType>
-    <BuildOS>$(__BuildOS)</BuildOS>
-    <Configuration Condition="'$(Configuration)' ==''">$(BuildType)</Configuration>
-    <Platform Condition="'$(Platform)'==''">$(BuildArch)</Platform>
-  </PropertyGroup>
+  <Import Project="..\dir.sdkbuild.props" />
```

Comment:
Should this be included in the `dirs.props` or is there a specific reason a new one was created?

Reply:
It's currently a bit confusing since the props files are in an intermediate state - some projects build using bulidtools and some against the SDK. dir.sdkbuild.props specifically has the common properties that were shared between the SDK projects and also the old projects. So it's currently not a good place to put props that apply **only** to SDK projects.

Reply:
I'll try to clarify this by adding some comments to the props files.

Reply:
I did some more clean-up of props files - see this commit message: https://github.com/dotnet/coreclr/pull/18695/commits/1d673026e7a459562e885b361cbb22e5d8dc969b.